src/main/java/frc/robot/commands/ReleaseRatchetCommand.java

```diff
@@ -5,37 +5,46 @@
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.subsystems.ShooterSubsystem;
-
-public class ShooterOffCommand extends CommandBase {
-  private ShooterSubsystem m_shooterSubsystem;
-
-  /** Creates a new ShooterOffCommand. */
-  public ShooterOffCommand(ShooterSubsystem shooterSubsystem) {
-    // Use addRequirements() here to declare subsystem dependencies.
-    m_shooterSubsystem = shooterSubsystem;
-    addRequirements(m_shooterSubsystem);
+import frc.robot.OperatorBoard.OperatorBoardButton;
+import frc.robot.subsystems.ClimberSubsystem;
+
+/**
+ * Releases ratchet while command is active. Command does NOT require climber
+ * subsystem. Locks ratchet while command is inactive.
+ */
+public class ReleaseRatchetCommand extends CommandBase {
+  private ClimberSubsystem m_subsystem;
+  private OperatorBoardButton m_button;
+
+  /** Creates a new ReleaseRatchetCommand. */
+  public ReleaseRatchetCommand(ClimberSubsystem subsystem, OperatorBoardButton button) {
+    m_subsystem = subsystem;
+
+    m_button = button;
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
+    m_subsystem.releaseRatchet();
+    m_button.turnLightOn();
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    m_shooterSubsystem.setFlywheelPower(0);
   }
 
   // Called once the command ends or is interrupted.
   @Override
   public void end(boolean interrupted) {
+    m_subsystem.lockRatchet();
```

Comment:
why are we locking the ratchet after releasing it? don't we want to extend the arm further?